### ISSUE TITLE
Project License

### ISSUE THREAD

**Issue by ld-cd** [Kedarius/RPi-CM4-Kicad#1]
Hi!

I'm a member of an avionics team for a college rocketry club planning on doing a CM4 based flight computer design. Most of our stuff is currently on Kicad 5.1.x so we don't want to upgrade to the nightly just yet. We try to be fairly careful about complying with license requirements and this repo currently doesn't have a license in it so I was wondering if you'd be willing to license under an open source license (BSD, GPL, etc) so we could use it in our design!

Best,
Aled

**Comment by Kedarius** [comment]
Hi,
I will update the repo as soon as I will get to my work computer. I am not exactly sure which license I want to use, probably BSD. I want it to be "do anything with the repo just don't blame me if your rocket failed because I swapped two pins :-)" sort of license....

**Comment by ld-cd** [comment]
Awesome!

Typically the BSD 3-clause contains an all caps no warranty section at the end so that should probably work (https://github.com/calstar/cas-rpi-hw/blob/master/LICENSE).

We also do a pretty thurough board review of everything we make, so if they'res anything swapped we will make a PR.

**Comment by Kedarius** [comment]
I've added the license information and also pushed some minor changes to the repo. The license thing is a little more complicated than I anticipated...
All my work is licensed under BSD 3-clause (as you suggested). That means the footprint and the schematic symbol. The symbol is completely made from scratch (well from the pin table from the datasheet), the footprint is taken from CM4IO-KiCAD.zip and modified for KiCad 5.x... The Raspberry Pi license for this is very open and there should be no problem with this.
The CM4 STEP model is taken directly from Raspberry Pi site with the same license so there should be no issue. 
I am not able to find the source of the HiRose connector STEP model but it can be downloaded from various sites. But for some reason the file directly from HiRose website is limited to non-commercial use and can not be redistributed....
In the new commit I also added the Raspberry Logo and that is trademarked stuff for sure. I am using it on the silkscreen so the user does put the CM4 board in the right way ("just align the Raspberry on the CM4 to the Raspberry on the silkscreen") but that would probably problematic from a trademark view....
